lib/menu_button.dart

```diff
@@ -59,7 +195,7 @@ class _MenuButtonState<T> extends State<MenuButton<T>> {
     final RenderBox overlay = Overlay.of(context).context.findRenderObject();
     final RelativeRect position = RelativeRect.fromRect(
       Rect.fromPoints(
-        button.localToGlobal(const Offset(0, 0), ancestor: overlay),
+        button.localToGlobal(const Offset(0, 12), ancestor: overlay),
```

Comment:
I'm not sure about the new Offset it should be `(0, 0)` to keep the Rect aligned
```suggestion
        button.localToGlobal(const Offset(0, 0), ancestor: overlay),
```

Reply:
I forgot change the 12, my mistake, sorry, but the 12 is the verticalContentPadding from LabelDecoration,  (if have a label decoration), if don't have a label decoration will be 0